packages/fxa-settings/src/pages/AccountRecoveryConfirmKey/en.ftl

```diff
@@ -0,0 +1,22 @@
+## AccountRecoveryConfirmKey page
+
+# Strings within the <span> elements appear as a subheading.
+# If more appropriate in a locale, the string within the <span>, "to continue to { $serviceName }" can stand alone as "Continue to { $serviceName }"
```

Comment:
```suggestion
# If more appropriate in a locale, the string within the <span>, "to continue to account settings" can stand alone as "Continue to account settings"
```

Reply:
If the service name is variable, should we still have "account settings" in the comment?

Reply:
Ohhhh but it's hard-coded in below, I understand now!